versions/3_6/requirements.txt

```
@@ -1,2 +1,2 @@
 grpcio-tools
-https://github.com/KodyPay/kody-clientsdk-python/releases/download/v0.1.8/kody_clientsdk_python-0.1.8.tar.gz
+https://github.com/KodyPay/kody-clientsdk-python/releases/download/v0.1.7/kody_clientsdk_python-0.1.7.tar.gz
```

Maybe we can make it consistent with 3.12 requirements ?

```suggestion
kody-clientsdk-python
```